frontend/src/toolbar/elements/heatmapToolbarMenuLogic.ts

```diff
@@ -141,7 +141,10 @@ export const heatmapToolbarMenuLogic = kea<heatmapToolbarMenuLogicType>([
                             date_to: values.commonFilters.date_to,
                         }
 
-                        defaultUrl = `/api/element/stats/${encodeParams({ ...params, paginate_response: true }, '?')}`
+                        defaultUrl = `/api/element/stats/${encodeParams(
+                            { ...params, paginate_response: true, sampling_factor: 0.1 },
```

Comment:
sampling was still in beta at posthog when we added sampling to elements API, let's allow the client to control it (but default to on)

checking in housewatch, switch to sampling factor isn't significantly faster or slower and is at least a consistent approach